mrbean/src/test/java/com/fasterxml/jackson/module/mrbean/TestBridgeMethods.java

```diff
@@ -0,0 +1,302 @@
+package com.fasterxml.jackson.module.mrbean;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.module.mrbean.AbstractTypeMaterializer.MyClassLoader;
+import org.objectweb.asm.AnnotationVisitor;
```

Comment:
In master, the package name has changed to `net.bytebuddy.jar.asm`

Reply:
Yes, we switched from Asm to ByteBuddy for 3.0... for tests we could still also include Asm unless you wanted to write alt implementation for BB.

Reply:
Asm was only replaced with ByteBuddy for 3.0 (see #28) so no need for that wrt tests.
Although since 2.x has lived longer than I thought back then, maybe backporting change for 2.13 would be good.
So maybe replacing `master` test from #106 would make sense.